weather-module.js

```diff
@@ -0,0 +1,20 @@
+var http = require('http')
+module.exports = function(apiKey, city, callback){
+  city = encodeURIComponent(city)
+  http.get({
+      host: 'api.openweathermap.org',
+      path: `/data/2.5/forecast/?q=${city}&APPID=${apiKey}&units=imperial`
```

Comment:
I'm torn here -- on the one hand, I'm totally down with students seeing this template string in the solution and having to Google for what it is. And they might recognize it from their work with Ruby. On the other hand, it could be a bit bewildering to them. Maybe we can just add a comment that points to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals?